ceph.spec.in

```diff
@@ -144,7 +144,12 @@ Requires:	python-rados = %{epoch}:%{version}-%{release}
 Requires:	python-rbd = %{epoch}:%{version}-%{release}
 Requires:	python-cephfs = %{epoch}:%{version}-%{release}
 Requires:	python-requests
+%if 0%{defined suse_version}
+Requires:	python-argparse
+%endif
+%if 0%{?rhel} || 0%{?centos} || 0%{?fedora}
```

Comment:
`0%{?rhel} || 0%{?centos}` can be simplified to just `0%{?rhel}`, since CentOS defines that macro for compatibility with RHEL.